crossfit/backend/dask/cluster.py

```diff
@@ -93,60 +92,6 @@ def increase_gc_threshold():
     gc.set_threshold(g0 * 3, g1 * 3, g2 * 3)
 
 
-def ensure_optimize_dataframe_graph(ddf=None, dsk=None, keys=None):
```

Comment:
Dask made these optimizations default  and this util should not be around

https://docs.dask.org/en/stable/changelog.html#legacy-dask-dataframe-implementation-removed